_targets.R

```diff
@@ -7,14 +7,26 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
-  tar_target(
-    site_data,
-    download_nwis_data(),
+  tar_target(site_data_01427207, download_nwis_site_data('01427207')),
+  tar_target(site_data_01432160, download_nwis_site_data('01432160')),
+  tar_target(site_data_01436690, download_nwis_site_data('01436690')),
+  tar_target(site_data_01466500, download_nwis_site_data('01466500')),
```

Comment:
tedious but the right way to do this for now! You will learn an easier way to do this in Pipelines 3 :)